### PR TITLE
Awesome books refactored with ES6

### PR DESCRIPTION
**Pull request by mrprotocoll**
Add Linters
Update README
Add Index.html
Add Index.js
Add Index.css
Add Modules
Add gitignore
Add Eslintignore

### CODE REVIEW

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,26 @@
+{
```
Comment (HAMMAS-SALEEM) [inline .eslintrc.json:1]: image

- [ ] You don't have a descriptive title for your pull request. :fearful: KIndly change Dev to something like ** Awesome books refactored with ES6** to make your pull request professional :+1:

File: README.md [hunk 2]
```diff
@@ -1 +1,158 @@
...
+
+- GitHub: [@mrprotocoll](https://github.com/mrprotocoll)
+- Twitter: [@dprotocoll](https://twitter.com/dprotocoll)
+- LinkedIn: [@mrprotocoll](https://www.linkedin.com/in/mrprotocoll)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+<!-- CONTRIBUTING -->
+
```
Comment (HAMMAS-SALEEM) [inline README.md:127]: - [ ] You don't have a Future Feature Section in your readme file :fearful: Kindly add it to make your readme descriptive and create some excitement for the viewer about the Future Features :+1:

File: README.md [hunk 3]
```diff
@@ -1 +1,158 @@
...
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 Awesome Book Project <a name="about-project"></a>
+
+> Describe of project.
+
```
Comment (HAMMAS-SALEEM) [inline README.md]: - [ ] Kindly remove this line :+1: to make your readme clean because this makes your readme look like a template :thinking:

**Review by HAMMAS-SALEEM — COMMENTED** [review]
# Invalid Submission :red_circle:

## Hey @mrprotocoll :wave:

## Highlights :1st_place_medal:

✔️ Descriptive PR Summary
✔️ Following some requirements

Good job on implementing some of the requirements but I cannot proceed review of your pull request because there are some issues that you still need to work on to go to the next project but you are almost there! :blush:

# Optional

_All the changes with optional tags are not crucial enough to prevent you from getting the approval but we highly recommend to students that they should take these changes into consideration._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

---------------------------------------

_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: .eslintrc.json (hunk 1 above)
Comment (mrprotocoll) [reply .eslintrc.json:1]: Thank You @HAMMAS-SALEEM , i will make the changes and request a new review

File: README.md (hunk 3 above)
Comment (mrprotocoll) [reply README.md]: Thank You @HAMMAS-SALEEM , i will make the changes and request a new review

**Review by omarsalem7 — APPROVED** [review]
Hi @mrprotocoll 😊, 
# STATUS: APPROVED! 🥇 💯 

giphy

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 🚀 
Congratulations! 🎉 

## Highlights
✅ Descriptive pull request 
✅ Project has met all the technical aspects you needed for this project. Keep rocking! 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**